thereadingmachine/modeller/lstm_k_step.py

```diff
@@ -3,55 +3,28 @@
 import pandas as pd
```

Comment:
Didn't we decide to drop the `lstm` process?

Reply:
Dropped from the process (i.e. `pipeline`) but we don't have to drop it from the code.